blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,143 @@
+import PropTypes from 'fusion:prop-types'
+import Consumer from 'fusion:consumer'
+import get from 'lodash/get'
+import moment from 'moment'
+import getProperties from 'fusion:properties'
+import { resizerKey } from 'fusion:environment'
+import { BuildContent } from '@wpmedia/feeds-content-elements'
+import { generatePropsForFeed } from '@wpmedia/feeds-prop-types'
+import { buildResizerURL } from '@wpmedia/feeds-resizer'
+
+const jmespath = require('jmespath')
+
+const cheerio = require('cheerio')
+
+const rssTemplate = (
+  elements,
+  {
+    channelTitle,
+    channelDescription,
+    channelCopyright,
+    channelTTL,
+    channelCategory,
+    channelLogo,
+    imageTitle,
+    imageCaption,
+    imageCredits,
+    itemTitle,
+    pubDate,
+    itemCategory,
+    itemDescription,
+    includePromo,
+    includeContent,
+    resizerURL,
+    resizerWidth,
+    resizerHeight,
+    domain,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  },
+) => ({
+  rss: {
+    '@version': '2.0',
+    channel: {
+      ...(channelTitle && { title: channelTitle }),
+      link: `${domain}`,
+      ...(channelDescription && { description: channelDescription }),
+      lastBuildDate: moment
+        .utc(new Date())
+        .format('ddd, DD MMM YYYY HH:mm:ss ZZ'),
+      ...(feedLanguage && { language: feedLanguage }),
+      ...(channelCategory && { category: channelCategory }),
+      ...(channelCopyright && {
+        copyright: channelCopyright,
+      }), // TODO Add default logic
+      ...(channelTTL && { ttl: channelTTL }),
+
+      ...(channelLogo && {
+        image: {
+          url: buildResizerURL(channelLogo, resizerKey, resizerURL),
+          title: channelTitle || feedTitle,
+          link: domain,
+        },
+      }),
+
+      item: elements.map((s) => {
+        let body, category
+        const url = `${domain}${s.website_url || s.canonical_url || ''}`
+        return {
+          title: jmespath.search(s, itemTitle) || '',
+          link: url,
+          guid: s._id,
+          pubDate: s[pubDate],
+          ...(s.content_elements &&  {
+            enclosure: {
+              '@href': `jmespath.search(s, 'content_elements[?type==audio].streams[].url[0]') || ''`,
```

Comment:
You are not using the customfield, you hard coded the string, so it is not configurable.
The example alexa feed uses a url attribute, not href
You should only generate an enclosure if you find a url.  Look at L80 below for an example of how to only generate a tag if you find data.

Reply:
You are not using the customfield, you hard coded the string, so it is not configurable.
> How we can make it configurable? I tried to access audioAvailable in customField but it's giving me different value like below:
{ bitrate: 5400, stream_type: 'mp3' }